Kronos-Shifts-Connector/Deployment/azuredeploy.json

```diff
@@ -587,66 +603,80 @@
             ]
         },
         {
-            "name": "[variables('keyVaultName')]",
             "type": "Microsoft.KeyVault/vaults",
-            "apiVersion": "2016-10-01",
+            "apiVersion": "2018-02-14",
+            "name": "[variables('keyVaultName')]",
             "location": "[parameters('location')]",
+            "comments": "This Azure KeyVault will store the necessary information like the Kronos credentials, etc.",
+            "dependsOn": [
+                "[concat('Microsoft.Web/sites/', variables('apiAppName'))]",
+                "[concat('Microsoft.Web/sites/', variables('configAppName'))]"
+            ],
+            "tags": {
+                "displayName": "KeyVault"
+            },
             "properties": {
-                "sku": {
-                    "family": "A",
-                    "name": "Standard"
-                },
+                "enabledForDeployment": "false",
+                "enabledForTemplateDeployment": "false",
+                "enabledForDiskEncryption": "false",
                 "tenantId": "[subscription().tenantId]",
-                "enabledForTemplateDeployment": true,
                 "accessPolicies": [
+                    {
+                        "objectId": "[reference(concat('Microsoft.Web/sites/', variables('apiAppName')), '2018-02-01', 'Full').identity.principalId]",
+                        "tenantId": "[subscription().tenantId]",
+                        "permissions": {
+                            "secrets": [
+                                "get"
+                            ],
+                            "keys": [
+                            ],
+                            "certificates": [
+                            ]
+                        }
+                    },
+                    {
+                        "objectId": "[reference(concat('Microsoft.Web/sites/', variables('configAppName')), '2018-02-01', 'Full').identity.principalId]",
+                        "tenantId": "[subscription().tenantId]",
+                        "permissions": {
+                            "secrets": [
+                                "get", "list", "delete", "set"
```

Comment:
Why does this need list?

Reply:
This has been removed as the "list" permission is not required